terraform/evaltree_provider.go

```diff
@@ -48,7 +35,6 @@ func ProviderEvalTree(n *NodeApplyableProvider, config *configs.Provider) EvalNo
 		},
 	})
 
-	// Apply stuff
```

Comment:
This comment (and the one above) did not seem to mean anything. I'd happily add more comments here if I knew what to write…